docs/tasks.rst

```diff
@@ -2,6 +2,14 @@
 Tasks Reference
 ==========================================
 
+activate_process_builder
+==========================================
+
+**Description:** Takes a selected list of developer names and runs activating flows against these orgs
```

Comment:
Hmm, this doesn't match what I see above. `docs/tasks.rst` should be generated from the actual code by running `cci task doc > docs/tasks.rst`, not edited by hand.